### ISSUE TITLE
Upgrade nested pages migrator to 0.7.4

### ISSUE THREAD

**Issue by mouhb** [xwikisas/application-confluence-migrator#95]
This [NPMIG-66](https://jira.xwiki.org/browse/NPMIG-66) was recently fixed on the nested pages migrator application, the fix has been backported to the "stable-0.7.x" branch used by the confluence migrator.
The objective of this ticket is to upgrade to 0.7.4 the dependency to nested pages application on the confluence migrator to benefit from the NPMIG-66 fix.

**Comment by mouhb** [comment]
On the first step of the confluence upgrade it is mentioned on the prerequisites that it is recommended to disable Solr Listeners, see screenshot below. This is a mistake because disabling Solr listener may cause pages backlinks update issues because backlinks are stored in Solr, see [NPMIG-66](https://jira.xwiki.org/browse/NPMIG-66).

The UI should be updated in order to remove the recommandation and replace it with another recommandation to ensure that the Solr queue is empty before starting the migration in order to minimize migration duration.

![upgrade-prerequisites-before](https://user-images.githubusercontent.com/3015046/222470973-06dfb6cf-2368-4c6c-b468-58575aeea1e6.png)

**Comment by mouhb** [comment]
* I update the project pom in order to upgrade the version of nested pages migrator (nestedpages.migrator.version) from 0.7.3 to 0.7.4
*  I updated the prerequisites content to update Solr recommendation.
* By the way I removed the cache requirement (_Set cache (xwiki.store.cache.capacity) to 5000 in xwiki.cfg (instead of 500)_) which is not needed anymore according to @snazare.

See on the below screenshot the UI after the modification.

![Uploading migration-prerequisites-after.png…]()
